build.gradle

```diff
@@ -255,6 +255,8 @@ dependencies {
     compile 'org.apache.commons:commons-math3:3.5'
     compile 'org.apache.commons:commons-collections4:4.1'
     compile 'org.apache.commons:commons-vfs2:2.0'
+    compile 'org.apache.commons:commons-configuration2:2.4'
+    compile 'commons-beanutils:commons-beanutils:1.9.3'
```

Comment:
Can you migrate to Owner instead of adding a new config library? I've suggested an approach that would allow you to use Owner in this ticket: #5963

Reply:
After offline discussions, we are not going to remove this dependency.  It can be done in another PR.

No action.